hydroflow/tests/datalog_frontend.rs

```diff
@@ -463,3 +492,34 @@ pub async fn test_join_multiple_then_relation() {
     assert_eq!(out_recv.next().await.unwrap(), (1, 2, 4, 5));
     assert_eq!(out_recv.next().await, None);
 }
+
+#[test]
+pub fn test_next_tick() {
+    let (ints_1_send, ints_1) = hydroflow::util::unbounded_channel::<(usize,)>();
+    let (ints_2_send, ints_2) = hydroflow::util::unbounded_channel::<(usize,)>();
+    let (result, mut result_recv) = fake_output_channel::<(usize,)>();
```

Comment:
For other tests we've been using `unbounded_channel` and `hydroflow::util::collect_ready`

```rust
    use hydroflow::util::collect_ready;
    let (result, mut result_recv) = hydroflow::util::unbounded_channel::<(usize,)>();
    // ...
    assert_eq!(&[(1,), (2,)], &*collect_ready::<Vec<_>, _>(&mut result_recv));
```